subprojects/core/src/main/java/org/gradle/plugin/management/internal/autoapply/AutoAppliedBuildScanPlugin.java

```diff
@@ -24,9 +26,20 @@
  *
  * Required by the {@code kotlin-dsl} module.
  */
-public interface AutoAppliedBuildScanPlugin {
-    PluginId ID = new DefaultPluginId("com.gradle.build-scan");
-    String GROUP = "com.gradle";
-    String NAME = "build-scan-plugin";
-    String VERSION = "1.16";
+public final class AutoAppliedBuildScanPlugin {
+
+    public static final PluginId ID = new DefaultPluginId("com.gradle.build-scan");
+    public static final String GROUP = "com.gradle";
+    public static final String NAME = "build-scan-plugin";
+    public static final String VERSION = "1.16";
+
+    /**
+     * Adds the {@code build-scan} plugin spec to the given {@link PluginDependenciesSpec} and returns the
+     * created {@link PluginDependencySpec}.
+     *
+     * @see PluginDependenciesSpec#id(String)
+     */
+    public static PluginDependencySpec addBuildScanPluginDependencySpecTo(PluginDependenciesSpec plugins) {
```

Comment:
I don't know if gradle have to support Java < 8 but you could also add just a `static` method to an java interface 🙃